druntime/src/core/sys/posix/sys/types.d

```diff
@@ -87,30 +87,21 @@ uid_t
 
 version (linux)
 {
-  static if ( __USE_FILE_OFFSET64 )
-  {
-    alias long      blkcnt_t;
-    alias ulong     ino_t;
-    alias long      off_t;
-  }
-  else
-  {
-    alias slong_t   blkcnt_t;
-    alias ulong_t   ino_t;
-    alias slong_t   off_t;
-  }
-    alias slong_t   blksize_t;
-    alias ulong     dev_t;
-    alias uint      gid_t;
-    alias uint      mode_t;
-    alias ulong_t   nlink_t;
-    alias int       pid_t;
```

Comment:
better to keep the whitespacing for these items (line 102-110), no change = better :)